libafl/src/generators/nautilus.rs

```diff
@@ -86,6 +87,14 @@ impl NautilusContext {
     /// Create a new [`NautilusContext`] from a file
     #[must_use]
     pub fn from_file<P: AsRef<Path>>(tree_depth: usize, grammar_file: P) -> Self {
+        if grammar_file.as_ref().extension().unwrap_or_default() == "py" {
+            let ctx = python_grammar_loader::load_python_grammar(
+                fs::read_to_string(grammar_file)
+                    .expect("Error reading grammar file")
```

Comment:
In general we should return Err here, not expect. The fn was copy&pasted but now that we touch it we might as well fix it..